src/main/java/uk/nhs/hee/tis/trainee/sync/service/TcsSyncService.java

```diff
@@ -84,6 +84,9 @@ public class TcsSyncService implements SyncService {
   private static final String REQUIRED_ROLE = "DR in Training";
   private static final String[] REQUIRED_NOT_ROLES = {"Placeholder", "Dummy Record"};
 
+  private static final Operation[] UPDATE_OPERATIONS
+      = {Operation.UPDATE, Operation.INSERT, Operation.LOAD};
```

Comment:
Looking at the operations in the existing SQS messages, it seems we need to monitor all of these.

Reply:
Yes, that's right - that's what you get when you let Sonar 'fix' the issue without thinking about actually fixing the issue 🤦